src/templates/kit/screens/ReviewScreen.js

```diff
@@ -18,8 +23,35 @@ function Review({
     next(data)
   }, [data, next])
 
+  const containerRef = useRef()
+  const prevNextRef = useRef()
+
+  useEffect(() => {
+    if (containerRef.current) {
+      containerRef.current.focus()
+    }
+  }, [])
+
+  useKeyDown(KEY_ENTER, () => {
+    // Don’t focus too early or the button will get clicked
+    setTimeout(() => {
+      if (
+        document.activeElement === containerRef.current &&
+        prevNextRef.current
+      ) {
+        prevNextRef.current.focusNext()
```

Comment:
Is this a react thing? I could not find documentation on it, could you share some?

Reply:
Just a custom ref object :-) https://github.com/aragon/aragon/blob/template-screens-tweaks/src/templates/kit/Navigation.js#L22

Reply:
:( that's a broken link